### ISSUE TITLE
Custom fish make vanilla fish exceedingly rare

### ISSUE THREAD

**Issue by Roxforbraynz** [MatrexsVigil/harvestcraft#148]
The custom fish are weighted in such a way that they crowd out the vanilla fish, making them very time-consuming to catch. This causes several problems with other mods that specifically need vanilla fish, like Botania's Ring of Chordata which specifically needs a raw salmon and a pufferfish. This also obstructs a vanilla mechanic; the Potion of Water Breathing, which needs a pufferfish to make. To put it into perspective, it took me close to an hour to catch a pufferfish for water breathing potions so I could work on an underwater base due to pufferfish being semi-rare to begin with.

**Comment by bennyman123abc** [comment]
If you do not like the custom fish and/or are having problems with it, you should disable all of the fish in the config for this mod.

**Comment by Roxforbraynz** [comment]
@bennyman123abc That was the first thing that crossed my mind. There was no config option to disable or change them in any way, thus the issue.

**Comment by Draconvarie** [comment]
Yes. If this could be fixed, it would be great.  Especially since the recipes that call for raw fish will only let me use vanilla raw fish. I am using MC version 10.2 and Forge 12.18.1.2011

**Comment by bennyman123abc** [comment]
I submitted a pull request that should hopefully fix this issue. If not, break the mod entirely. I was not able to test the mod as I have no way to compile the source right now (Modding workstation is broken) but in theory, my changes should work. #170 #169
